src/UTILS/utils.js

```diff
@@ -4,8 +4,14 @@ const ncNews = axios.create({
   baseURL: "https://nc-news-sarah.onrender.com/api",
 });
 
-export const getArticles = () => {
-    return ncNews.get('/articles').then((response) => {
+export const getArticles = (topic) => {
+  let url=""
+  if (!topic) {
+    url = '/articles'
+  } else if (topic) {
+    url = `/articles?topic=${topic}`
+  }
```

Comment:
Cool addition of logic to add an optional query to the api request
This could also be achieved using axios's params functionality https://notes.northcoders.com/courses/js-front-end/axios#axios-params